.github/workflows/build-preview-mirror.yml

```diff
@@ -39,5 +39,12 @@ jobs:
     uses: ./.github/workflows/integrity-check.yml
     with:
       repo: ${{ inputs.repo || 'https://preview-mirror.mage-os.org/' }}
-    secrets: 
-      composer_auth: ${{ secrets.composer_auth }}
\ No newline at end of file
+    secrets:
+      composer_auth: ${{ secrets.composer_auth }}
+  integrity-check-package-versions-validate:
```

Comment:
This shouldn't be added here separately, it should be inside the other integrity check workflow.